docs/source/tutorials_source/platform/tutorial_active_learning_detectron2.py

```diff
@@ -0,0 +1,306 @@
+cfg.merge_from_file(model_zoo.get_config_file("COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml"))
+cfg.MODEL.ROI_HEADS.SCORE_THRESH_TEST = 0.5  # set threshold for this model
+# Find a model from detectron2's model zoo. You can use the https://dl.fbaipublicfiles... url as well
+###cfg.MODEL.WEIGHTS = model_zoo.get_checkpoint_url("COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml")
+cfg.MODEL.WEIGHTS = model_zoo.get_checkpoint_url("COCO-Detection/faster_rcnn_R_50_FPN_3x.yaml")
+predictor = DefaultPredictor(cfg)
+
+# %%
+# We use this little helper method to overlay the model predictions on a
+# given image.
+def predict_and_overlay(model, filename):
+    # helper method to run the model on an image and overlay the predictions
+    im = cv2.imread(filename)
+    out = model(im)
+    # We can use `Visualizer` to draw the predictions on the image.
+    v = Visualizer(im[:, :, ::-1], MetadataCatalog.get(cfg.DATASETS.TRAIN[0]), scale=1.2)
+    out = v.draw_instance_predictions(out["instances"].to("cpu"))
+    plt.figure(figsize=(16,12))
+    plt.imshow(out.get_image()[:, :, ::-1])
+    plt.axis('off')
+    plt.tight_layout()
+
+# %%
+# The lightly framework expects a certain bounding box and prediction format. 
+# We create another helper method to convert the detectron2 output into 
+# the desired format.
+def convert_bbox_detectron2lightly(outputs):
+    # convert detectron2 predictions into lightly format
+    height, width = outputs['instances'].image_size
+    boxes = []
+
+    for (bbox_raw, score, class_idx) in zip(outputs['instances'].pred_boxes.tensor, outputs['instances'].scores, outputs['instances'].pred_classes):
+        x0, y0, x1, y1 = bbox_raw.cpu().numpy()
+        x0 /= width
+        y0 /= height
+        x1 /= width
+        y1 /= height
+      
+        boxes.append(BoundingBox(x0, y0, x1, y1))
+    output = ObjectDetectionOutput.from_scores(boxes, outputs['instances'].scores.cpu().numpy(), outputs['instances'].pred_classes.cpu().numpy().tolist())
+    return output
+
+# %%
+# Run model predictions
+# ----------------------
+#
+# We now use the created model and iterate over the `query_set` and make predictions.
+# It's important that the predictions are in the same order as the filenames
+# in the `query_set`. Otherwise, we could upload a prediction to the wrong sample!
+
+obj_detection_outputs = []
+pbar = tqdm.tqdm(al_agent.query_set)
+for fname in pbar:
+  fname_full = os.path.join(DATASET_ROOT, fname)
+  im = cv2.imread(fname_full)
+  out = predictor(im)
+  obj_detection_output = convert_bbox_detectron2lightly(out)
+  obj_detection_outputs.append(obj_detection_output)
+
+# %%
+# Now, we need to turn the predictions into scores.
+# The scorer assigns scores between 0.0 and 1.0 to 
+# each sample and for each scoring method.
+scorer = ScorerObjectDetection(obj_detection_outputs)
+scores = scorer.calculate_scores()
+# %% 
+# Let's have a look at the sample with the highest
```

Comment:
It's now called uncertainty-margin so you should also adapt the text.

I would also appreciate a short box here (maybe a note) about what it means to have a high uncertainty-margin.

E.g.

Note: A high uncertainty margin means that the image contains at least one bounding box for which the model is unsure about the class of the object in the bounding box. Read more about how our active learning scores are calculated [here](link).